server/common/transport-shared/src/main/java/org/kaaproject/kaa/server/transport/message/ErrorBuilder.java

```diff
@@ -13,9 +13,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.kaaproject.kaa.server.operations.service.akka.messages.io.request;
+package org.kaaproject.kaa.server.transport.message;
 
+/**
+ * Converts exception into objects specific to corresponding transport channel
```

Comment:
Converts an exception into objects specific to the corresponding transport channel